parse/TechsParser.cpp

```diff
@@ -116,14 +116,14 @@ namespace {
 
             prerequisites
                 =   parse::label(Prerequisites_token)
-                >  (    '[' > +tok.string [ insert(_r1, _1) ] > ']'
+                >  (    ('[' > +tok.string [ insert(_r1, _1) ] > ']')
```

Comment:
Nor do I... I'd probably have added some if I did (and actually, I probably have in a few places). But I'd rather add brackets than deal with disabling warnings and resulting quirks or maintenance problems.